app.js

```diff
@@ -25,18 +28,21 @@ const sessionSettings = {
 // register all partials from directory.
 hbs.registerPartials("./views/partials");
 
-app.set("views", "./views");
+app.set("views", path.join(__dirname, "views"));
```

Comment:
Right, but you're not being consistent with the usages here. You're `require()`-ing from `./` anyway, so it'll break regardless.